<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 77, 77);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>40</number>
    </property>
    <item>
     <widget class="QWidget" name="mainmenu_widget" native="true">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(106, 106, 106);
background-color: qlineargradient(spread:pad, x1:0.534, y1:0, x2:0.563, y2:1, stop:0 rgba(42, 0, 79, 255), stop:1 rgba(0, 0, 0, 255));</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="welcome_label">
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;

</string>
          </property>
          <property name="text">
           <string>Welcome!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Please press any of the buttons below to start</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="GeometryEdit_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(162, 162, 162);
color: &quot;black&quot;
}

QPushButton:hover {
background-color: rgb(38, 186, 255);
}

QPushButton:pressed {
	background-color: rgb(19, 92, 126);
}</string>
          </property>
          <property name="text">
           <string>Geometry Editing</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ProjectorConfig_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(162, 162, 162);
color: &quot;Black&quot;
}

QPushButton:hover {
background-color: rgb(38, 186, 255);
}

QPushButton:pressed {
	background-color: rgb(19, 92, 126);
}</string>
          </property>
          <property name="text">
           <string>Projector Configuration</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="Add_Projector_action">
   <property name="text">
    <string>Add Projector</string>
   </property>
  </action>
  <action name="Remove_Projector_action">
   <property name="text">
    <string>Remove Projector</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
